src/compiler/emitter.ts

```diff
@@ -2399,7 +2399,7 @@ namespace ts {
 
         function emitEnumMember(node: EnumMember) {
             emit(node.name);
-            emitInitializer(node.initializer);
+            emitInitializer(node.initializer, node.name.end + 1, node);

```

Comment:
Why are these all plus one? Shouldn't the start position be the end of the name?

Reply:
Yep. This one (and like one other?) were the first ones I wrote and for some reason thought that I needed to skip the length the token in question. Thanks for reminding me that I still hadn't gone back and changed them.

Reply:
Maybe that's a good indicator that we need tests for cases where trivia comes immediately after: